[DOC]: Missing DevKit installation in CONTRIBUTING file

### What's wrong?

Following the [CONTRIBUTING.md](https://github.com/SuperDuperDB/superduperdb/blob/main/CONTRIBUTING.md) fails when trying to execute `make unit_testing`:

1. Get the code on your local:

```shell
# Clone and change location directory of the superduperdb repository, change the `<FORKED_NAME>` to your GitHub id
git clone git@github.com:<FORKED_NAME>/superduperdb.git
cd superduperdb
```

2. Set up your python environment:

```shell
# Create your Python virtual environment
python3 -m venv .venv

# Activate the Python virtual environment
. .venv/bin/activate  
```

3. Install the dependencies:

```shell
# Install pip-tools and latest version of pip
python3 -m pip install --upgrade pip-tools

# Install the superduperdb project in editable mode along with the developer tools
python3 -m pip install -e '.'
python3 -m pip install -r deploy/installations/testenv_requirements.txt
```

4. Run unit tests:

```shell
$ make unit_testing
pytest  ./test/unittest/
make: pytest: No such file or directory
make: *** [unit_testing] Error 1
```


### How could it be better?

Including the `make install_devkit` command in the setup instructions would ensure a smoother setup and ensure more consistent development environments 🙂🚀

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -48,6 +48,7 @@ python3 -m pip install --upgrade pip-tools
 # Install the superduperdb project in editable mode along with the developer tools
 python3 -m pip install -e '.'
 python3 -m pip install -r deploy/installations/testenv_requirements.txt
+make install_devkit
 ```
 
 (Optional) build the docker development environment:

